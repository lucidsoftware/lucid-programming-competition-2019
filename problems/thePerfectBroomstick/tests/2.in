2
11
3
9
20
9
9
15
9
6
17
20
14
14
17
12
14
5
12
15
2
11
17
2
9
16
11
8
19
4
6
6
2
3
13
16
10
10
19
16
19
8
12
17
17
4
9
10
4
16