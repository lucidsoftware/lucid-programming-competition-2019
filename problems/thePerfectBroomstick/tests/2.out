2
11
3
9
9
9
9
9
9
9
9
9
9
11
11
12
12
12
12
12
12
12
12
12
11
12
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
11
